### PR TITLE
III-5111 Refactor Unit to avoid deprecated methods

### PR DESCRIPTION
**Pull request by JonasVHG**
### Fixed
- replaced deprecated `assertFileNotExists()` with `assertFileDoesNotExist()`
- replaced deprecated `at()` depending on how it was used.
- Code cleanup of edited files

---
Ticket: https://jira.uitdatabank.be/browse/III-5111

### CODE REVIEW

File: tests/Offer/BulkLabelCommandHandlerTest.php [hunk 1]
```diff
@@ -149,15 +149,13 @@ public function it_logs_an_error_when_an_error_occurred_and_continues_labelling(
...
-        $this->commandBus->expects($this->at(0))
-            ->method('dispatch')
-            ->with(new AddLabel($this->offerIdentifiers[1]->getId(), $this->label))
-            ->willThrowException($exception);
-
         // Make sure the other offer is still labelled.
-        $this->commandBus->expects($this->at(1))
-            ->method('dispatch')
-            ->with(new AddLabel($this->offerIdentifiers[2]->getId(), $this->label));
+        $this->commandBus->method('dispatch')->will(
```
Comment (LucWollants) [inline tests/Offer/BulkLabelCommandHandlerTest.php:147]: Is it intentional to no longer test the parameters?

**Review by LucWollants — APPROVED** [review]
I only had a few smaller remarks, so I already approved. Kudos for the extra boy scouting.

File: tests/Offer/BulkLabelCommandHandlerTest.php (hunk 1 above)
Comment (JonasVHG) [reply tests/Offer/BulkLabelCommandHandlerTest.php:147]: The trouble, as far as I could find, is that on `withConsecutive()` or `willReturnMap()` you cannot throw an exception the first time it's run and than continue.

Comment (LucWollants) [reply tests/Offer/BulkLabelCommandHandlerTest.php:147]: I think you can add this
```
            ->withConsecutive(
                [new AddLabel($this->offerIdentifiers[1]->getId(), $this->label),],
                [new AddLabel($this->offerIdentifiers[2]->getId(), $this->label),]
            )
```

Comment (JonasVHG) [reply tests/Offer/BulkLabelCommandHandlerTest.php]: Thanks, that's the trick🪄

**Review by LucWollants — APPROVED** [review]

**Pull request merged**